etc/links.profile

```diff
@@ -0,0 +1,59 @@
+# Firejail profile for links
+# Description: Text WWW browser
+# This file is overwritten after every install/update
+#
+# Persistent local customizations
+include links.local
+# Persistent global definitions
+# include globals.local
+
+noblacklist ${HOME}/.links
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.links
+whitelist ${HOME}/.links
+whitelist ${DOWNLOADS}
+
+caps.drop all
+ipc-namespace
+# uncomment machine-id if you want to allow access only to user-configured
+# associated music player
+machine-id
+netfilter
+# uncomment no3d if you want to allow access only to user-configured
+# associated video player
+no3d
+nodvd
+nogroups
+nonewprivs
+noroot
+# uncomment nosound if you want to allow access only to user-configured
+# associated music player
+nosound
+notv
+nou2f
+# uncomment novideo if you want to allow access only to user-configured
+# associated video player
+novideo
+protocol unix,inet,inet6
+seccomp
+shell none
+tracelog
+# uncomment to allow user-configured associated programs to use X11
+x11 none
+
+disable-mnt
+private-cache
+private-dev
+private-tmp
+# uncomment if you want to allow access only to user-configured associated programs
+# private-bin links,sh,PROGRAM1,PROGRAM2,PROGRAM3
```

Comment:
Keep oder and put above `private-cache` also I think it is better to enable it like: 
```
# if you want to use user-configured programs add 'private-bin PROGRAM1,PROGRAM2' to your links.local
# or append 'PROGRAM1,PROGRAM2' to this private-bin line
private-bin links,sh
```